app/views/grant_permission_mailers/submission_mailer/submitted_notification.html.haml

```diff
@@ -5,8 +5,8 @@
   has a new submission for Grant:
   #{ link_to @grant.name, grant_url(@grant) }.
 %p
-  Applicant:
-  = full_name(@applicant)
+  Submitter:
```

Comment:
Add logic to display both applicant and submitter info if the two are different?

Reply:
I think both should always be displayed here. I did shift Applicants up and pluralized it because I think this cleans up the display and puts the important person at the top. 

I would either remove submitter all together or leave it in even if it is the same.

Reply:
@lmkw for outgoing submission notification emails where the submitter and the applicant are the same person, does it still make sense to include both "Submitted by: John Doe" and "Applicant: John Doe" in the outgoing notification email or is it sufficient to just show "Applicant"? Should "Submitted by" info be included at all?

Reply:
Although I see Matt's point about consistency, I think it can also be visually noisy to have both listed when they are the same person. So I would go for, prioritize Applicant and only show Submitter when Submitter is different from Applicant


So this is easier to read:

Applicant: Dr. Bunsen Honeydew    Submitter: Beaker

Applicant: Kermit

Applicant: Rolf

Applicant: Animal   Submitter: Janice



than this:

Applicant: Dr. Bunsen Honeydew    Submitter: Beaker

Applicant: Kermit   Submitter: Kermit

Applicant: Rolf    Submitter: Rolf

Applicant: Animal   Submitter: Janice